src/com/magento/idea/magento2plugin/actions/generation/dialog/NewModuleDialog.java

```diff
@@ -72,6 +73,8 @@
     private JLabel moduleDescriptionLabel;//NOPMD
     private JLabel moduleNameLabel;//NOPMD
 
+    private static final String MAGENTO_OLD_VERSION = "2.2.11";
```

Comment:
Let it be `MAGENTO_BEFORE_DECLARATIVE_SCHEMA_VERSION`. What do you think?

Reply:
Yep, It's better.